src/test/run-pass/generator/yield-in-args-rev.rs

```diff
@@ -12,12 +12,10 @@
 
 fn foo(_a: (), _b: &bool) {}
 
-// Some examples that probably *could* be accepted, but which we reject for now.
-
 fn bar() {
```

Comment:
I think a comment would be good here to explain the purpose of this test.

For example:

```
// Test that a yield occuring in the list of arguments *before* a borrow
// does not cause a borrowck error about having a live borrow.
```

Sorta obvious from the name but still useful sometimes.